parser/html_to_dita.py

```diff
@@ -111,6 +111,8 @@ def htmlToDITA(soup_in, dita_soup, div_replacement="span", wrap_strings=False):
                 # TODO: examine use of centre-aligned DIVs. Do we need to reproduce that formatting?
                 del div["align"]
                 del div["style"]
+        if div.get("id") == "":
```

Comment:
@robintw - is this the fix to a specific issue?  There was a div yesterday that mistakenly had an empty string as id (abbreviations).  But, I don't know if there are any more in the data that still need to be fixed.  Could we please also log a warning when we delete content like this?  (since it could be destructive without us noticing it).

Reply:
Yeah this is to deal with that div in Abbreviations. We won't actually be deleting the div itself, we're just deleting the id value, so I wasn't sure if that counted as deleting content?

Reply:
aah yeah - we're deleting the id, not the whole element.  This fix is fine :-)